RELEASE_NOTES.md

```diff
@@ -116,14 +105,11 @@ making a backup is strongly advised.
 
   none             : use neither base-curve nor filmic
 
-- Filmic RGB goes to v4 (new color science) with integrated highlight recovery.
+- Filmic RGB is updated to v4 (new color science) with integrated highlight recovery.
 
 ## New Features And Changes
 
-- Add support for curved gradients. The gradient mask can now be
-  curved to revert fish-eye kind distortion and so become a straight
-  line on the horizon for example. This can also be used to artistic
-  goals.
+- Add support for curved gradients. This can be helpful when putting a gradient mask on an  image with a horizon line that is curved because lens distortion. This can also be used towards artistic goals.
```

Comment:
"...curved **due to** lens distortion..."
"...can also be used **for** artistic goals."